front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -90,6 +92,48 @@ public Collection<Pipeline> list(
     return pipelineDAO.all(refresh);
   }
 
+  /**
+   * Get all pipelines triggered after a pipeline executes.
+   *
+   * @param id the id of the completed/triggering pipeline
+   * @param status the execution status of the pipeline (canceled/suspended/succeeded)
+   */
+  @PreAuthorize("#restricted ? @fiatPermissionEvaluator.storeWholePermission() : true")
+  @PostFilter("#restricted ? hasPermission(filterObject.name, 'APPLICATION', 'READ') : true")
+  @RequestMapping(value = "triggeredBy/{id:.+}/{status}", method = RequestMethod.GET)
+  public Collection<Pipeline> getTriggeredPipelines(
+      @PathVariable String id,
+      @PathVariable String status,
+      @RequestParam(required = false, value = "restricted", defaultValue = "true")
+          boolean restricted,
+      @RequestParam(required = false, value = "refresh", defaultValue = "true") boolean refresh) {
+
+    Collection<Pipeline> pipelines = pipelineDAO.all(refresh);
+
+    Predicate<Trigger> triggerPredicate =
+        trigger -> {
+          boolean retval =
+              trigger.getEnabled()
+                  && (trigger.getType().equals("pipeline"))
+                  && id.equals(trigger.getPipeline())
+                  && trigger.getStatus().contains(status);
+          log.debug(
+              "pipeline configuration id {} with status {} {} trigger {}",
+              id,
+              status,
+              retval ? "matches" : "does not match",
+              trigger.toString());
+          return retval;
+        };
+
+    // Return all pipelines with at least one trigger defined for the given id
+    // and status.
+    Predicate<Pipeline> pipelinePredicate =
```

Comment:
This is supposed to be equivalent to [this logic in orca](https://github.com/spinnaker/orca/blob/da3cf93de4f003a91d3ddd01c1996d0a8961e58f/orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/spring/DependentPipelineExecutionListener.groovy#L93-L101), except for the difference highlighted [here](https://github.com/spinnaker/front50/pull/1251/files#diff-ef16150a5e2666115561825274021544fe6e234f34b5cd5f7bd29c67469afa49R41).